dask_match/tests/test_collection.py

```diff
@@ -222,3 +222,35 @@ def test_copy(pdf, df):
 
     assert tuple(original.columns) == columns
     assert "z" not in original.columns
+
+
+def test_partitions(pdf, df):
+    assert_eq(df.partitions[0], pdf.iloc[:10])
+    assert_eq(df.partitions[1], pdf.iloc[10:20])
+    assert_eq(df.partitions[1:3], pdf.iloc[10:30])
+    assert_eq(df.partitions[[3, 4]], pdf.iloc[30:50])
+    assert_eq(df.partitions[-1], pdf.iloc[90:])
+
+
+@pytest.mark.parametrize("ignore_index", [True, False])
+@pytest.mark.parametrize("npartitions", [None, 2])
+def test_simple_shuffle(ignore_index, npartitions):
+    df = from_pandas(
+        pd.DataFrame({"x": list(range(20)) * 5, "y": range(100)}),
+        npartitions=10,
+    )
+    df2 = df.shuffle("x", npartitions=npartitions, ignore_index=ignore_index)
+
+    # Check that the output partition count is correct
+    assert df2.npartitions == (npartitions or df.npartitions)
+
+    # Check the computed (re-ordered) result
+    assert_eq(df, df2.compute(), check_index=not ignore_index)
```

Comment:
I'm curious, why the compute call here?

Reply:
I’m just checking that the data is unchanged. assert_eq will sort the result, but will not repartition. Other ideas are very welcome.

Reply:
> assert_eq will sort the result, but will not repartition

These things seem ok?  Things seem to pass without the compute call.  In general I always try to give assert_eq the full collection so that it can run it's other checks.  I may not fully understand what's going on here though.

Reply:
Ah fair point - This was originally failing for me at one point for me, but probably for a different reason than I thought.